pages/posts/[postID].js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+import Post from "../../components/layout/postsDashboard/Post";
+import { useRouter } from "next/router";
+import { server } from "../../config/index";
+
+export async function getStaticPaths() {
```

Comment:
Is that needed? You call api endpoint and get info about post.

Reply:
If i'm not mistaken, regular fetching with useEffect() didn't work for me because of dynamic routing. So i searched for alternatives and found this

Reply:
Ok, fair enough